Util_Environment.php

```diff
@@ -966,7 +966,7 @@ public static function normalize_file_minify( $file ) {
 	 */
 	public static function url_to_docroot_filename( $url ) {
 		$data = array(
-			'home_url' => get_home_url(),
+			'home_url' => get_site_url(),
```

Comment:
my short guess is that if stable solution exists - it should change something further, after those url_to_docroot_filename.
also test environent with that case should be added, otherwise next fix will kill it.